server.js

```diff
@@ -15,19 +17,62 @@ import cors from 'cors'
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+const endPoints = require('express-list-endpoints')
+app.get("/", (request, response) => {
+  response.send(endPoints(app));
+});
+
+// Get all the books in the list & filter options
+app.get('/books', (req, res) => {
+  // get only books that have been ranked
+  const isRanked = req.query.isRanked
+
+  // get only books with a certain minimum ranking
+  const minRating = req.query.minRating;
+
+  // get only books by a certain author
+  const author = req.query.author;
```

Comment:
This could be destructured, like:
`const { isRanked, minRating, author } = req.query;`